src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Strings.resx

```diff
@@ -285,4 +285,8 @@
   <data name="Exception_InvalidContinuationToken" xml:space="preserve">
     <value>Invalid token</value>
   </data>
+  <data name="InvalidCollectionElementType" xml:space="preserve">
```

Comment:
This key has a name that looks like a general error when a collection has an element of the wrong type, but the message text is very specific to this one particular case. Can we rename this key so it matches the specificity of the error messsage?